cfgov/jinja2/v1/complaint/complaint-landing.html

```diff
@@ -68,12 +68,13 @@ <h1 class="m-hero_heading">Consumer Complaint Database</h1>
 
 {% block content_main %}
 
-<div class="m-notification m-notification__warning {% if technical_issues %} m-notification__visible show-data-notification {% endif %}">
+{% if technical_issues %}
+<div class="m-notification m-notification__warning m-notification__visible ">
```

Comment:
TRIVIAL: could be remove the ending whitespace after the end of the classes `m-notification__visible">`

Reply:
√